libs/watt/src/lib/configuration/watt-locale.service.ts

```diff
@@ -29,6 +30,15 @@ export class WattLocaleService {
   constructor(private dateAdapter: DateAdapter<unknown>) {}
 
   setActiveLocale(locale: WattSupportedLocales): void {
+    if (locale === 'da') {
+      import('dayjs/locale/da');
+    }
+
+    if (locale === 'en') {
+      import('dayjs/locale/en');
+    }
```

Comment:
Question: Why not make this dynamic? We already have type-safety for the locale since it should be a supported locale. So why not make it easier to add other locales and remove some if statements? :)
OR does AST come into play here?
```suggestion
     import(`dayjs/locale/${locale}`);
```

Reply:
I can try it out, but I do not thing it will work. It would have a hard time figuring out what dynamic imports to create when it cannot static analyze the code?

Reply:
![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1250254/9109e8ed-d7e3-4449-9527-129f34587183)

As I suspected